receiver/apachesparkreceiver/metadata.yaml

```
@@ -4,4 +4,783 @@ status:
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: "{ tasks }"
+    attributes:
+      [application_id, application_name, stage_id, attempt_id, stage_status]
+  spark.stage.killed_tasks:
+    description: Number of killed tasks in this stage.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: "{ tasks }"
+    attributes:
+      [application_id, application_name, stage_id, attempt_id, stage_status]
+  spark.stage.executor_run_time:
+    description: Amount of time spent by the executor in this stage.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: ms
+    attributes:
+      [application_id, application_name, stage_id, attempt_id, stage_status]
+  spark.stage.executor_cpu_time:
+    description: CPU time spent by the executor in this stage.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: ns
+    attributes:
+      [application_id, application_name, stage_id, attempt_id, stage_status]
+  spark.stage.result_size:
+    description: The sum of the bytes transmitted back to the driver by all the tasks in this stage.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: bytes
+    attributes:
+      [application_id, application_name, stage_id, attempt_id, stage_status]
+  spark.stage.jvm_gc_time:
+    description: The amount of time the JVM spent on garbage collection in this stage.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: bytes
+    attributes:
+      [application_id, application_name, stage_id, attempt_id, stage_status]
+  spark.stage.memory_spilled:
+    description: The amount of memory moved to disk due to size constraints (spilled) in this stage.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: bytes
+    attributes:
+      [application_id, application_name, stage_id, attempt_id, stage_status]
+  spark.stage.disk_space_spilled:
+    description: The amount of disk space used for storing portions of overly large data chunks that couldn’t fit in memory in this stage.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: bytes
+    attributes:
+      [application_id, application_name, stage_id, attempt_id, stage_status]
+  spark.stage.peak_execution_memory:
+    description: Peak memory used by internal data structures created during shuffles, aggregations and joins in this stage.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: bytes
+    attributes:
+      [application_id, application_name, stage_id, attempt_id, stage_status]
+  spark.stage.input_bytes:
```

Typically don't put the unit in the metric name. We have a dedicate field for units, and units are often converted (making the name misleading).
```suggestion
  spark.stage.input_size:
```